doc/source/extended_documentation/calibration/rainforests_calibration/rainforests_calibration.rst

```diff
@@ -27,86 +26,52 @@ a suitable set of meteorological variables can be used to distinguish different
+the final output.
+
+One advantage of the ECPoint approach is that the calibration is inherently non-local. As the calibration is done by
+identifying distinct weather types, the model bias and scale difference should be independent of any given location and time as 
+the underlying physical process should be identical. Thus a grid-point can be calibrated using data from any location, provided the 
+underlying weather type is the consistent. This enables effective calibration to be applied to areas that are typically lacking 
+sufficient cases to calibrate against.
+
 
 ****************************
 The RainForests method
 ****************************
 
-RainForests is an adaptation of the ECPoint method that seeks to use machine learning based
+RainForests is an adaptation of the ECPoint method that uses machine learning based
 tree models to replace the manually constructed tree model of ECPoint. Here we use gradient-boosted
-decision tree (GBDT) ensemble models.
-
-The underlying principle of the calibration methodology is essentially the same, namely using
-a set of feature variables to map onto error distributions which can be applied to the
-forecast values to produce calibrated forecast values. However, the way in which these
-distributions are constructed differs somewhat within the RainForests framework.
-
-Our approach is to use a series of GBDT models, taking the feature variables as inputs, to
-produce exceedance probability values for representative error thresholds. Collectively
-these represent the error distributions as a cumulative distribution function.
+decision tree (GBDT) models.
 
-Each CDF is then mapped onto a series of equispaced percentile values to yield a series of
-representative error values which can be applied to the grid-scale forecast value to produce
-the series of possible point-scale (calibrated) values.
+Our aim is to produce a probability distribution of the expected rainfall, given the NWP 
+forecasts of relevant variables. We define a set of rainfall thresholds, suitably spaced so as 
+to accurately model the distribution. Then we train a separate GBDT model for each lead time and 
+threshold (note that each GBDT model itself consists of several hundred trees).
 
-Calibration of the ensemble forecast proceeds by determining the underlying distribution on
-a per realization basis and applying this to each forecast value to produce a series of
-distinct calibrated forecast values for each realization. Collectively these values form the
-so-called super-ensemble, which we subsequently sample to produce the calibrated forecast
-ensemble output.
+The input variables for the calibration are generally the NWP forecasts of the variable of interest, 
+and related variables (for example, when calibrating precipitation, a related variable would 
+be convective precipitation), along with ancillary variables such as solar time.
```

Comment:
I think this is a good change, but I might suggest the following to be more focused:
```suggestion
The set of variables that feed into the GBDT models are what allows the calibration to distinguish between
different weather situations and build the appropriate probability distribution accordingly. These input
variables are typically diagnostic variables for the ensemble realization (including the variable of interest),
but can include static and dynamic ancillary variables, such as local solar time, and whole-of-ensemble
values for diagnostic variables, such as mean or standard-deviation.
```

Reply:
done